symfony/plugins/orangehrmTimePlugin/test/Api/MyTimesheetItemAPITest.php

```diff
@@ -31,9 +33,9 @@ class MyTimesheetItemAPITest extends EndpointIntegrationTestCase
      */
     public function testGetAll(TestCaseParams $testCaseParams): void
     {
-        $this->populateFixtures('MyTimesheetItemAPITest.yaml');
+        TestDataService::populate(Config::get(Config::TEST_DIR) . '/phpunit/fixtures/WorkflowStateMachine.yaml', true);
+        $this->populateFixtures('MyTimesheetItemAPITest.yaml',null,true);
```

Comment:
```suggestion
        $this->populateFixtures('MyTimesheetItemAPITest.yaml', null, true);
```
please format this line

Reply:
done

Reply:
done